backend/app/config.py

```diff
@@ -38,6 +38,7 @@ class Settings(BaseSettings):
     stripe_secret_key: Optional[str] = None
     stripe_public_key: Optional[str] = None
     stripe_webhook_key: Optional[str] = None
+    flat_manager_secret: str = "c2VjcmV0"
```

Comment:
This is probably a placeholder? Might be nicer, to have the default be something, that will cause a failure, so you know, when you've set it correctly/at all.

Reply:
If I set it to None here, is there a way to override it for development?

Reply:
There seems to be https://pydantic-docs.helpmanual.io/usage/settings/#environment-variable-names

But I don't think I'm a fan of that either, so let's keep it like it is.

Reply:
I override the stripe stuff for development using envvars, you can see how in the docker-compose.yml - it's fairly easy.  But also for this kind of thing, just having a simple "this is a test key" or somesuch as the key will do I think.

Reply:
This is "secret" in base64, same as the default in the flat-manager repo.